src/components/CvButton/CvIconButton.vue

```diff
@@ -30,8 +30,7 @@ const { disabled, icon, kind, size } = commonCvButtonProps;
 
```

Comment:
I'm getting this warning:
![image](https://github.com/carbon-design-system/carbon-components-vue/assets/20342514/348ef293-1506-4943-ba7a-880c9974ceda)

The value received is an empty string, but alsp seems this prop/attribute is not used in the story at all

Reply:
Updated story
1. Get tip alignments fom '../CvTooltip/consts.js';
2. Fix warnings from story book about deprecated format.
3. Add control for tipAlignment